src/models/tests/m_rsc_db_tests.erl

```diff
@@ -11,8 +11,8 @@ modify_rsc_test() ->
     AdminC = z_acl:logon(?ACL_ADMIN_USER_ID, C),
     CatId = m_rsc:rid(text, C),
 
-    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}], C)),
-    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}, {category_id, CatId}], C)),
+    ?assertThrow({{error, eacces}, _Trace}, m_rsc:insert([{title, "Hello."}], C)),
```

Comment:
Yes, that trace is wrong.

Btw, I don't like that throw anyway, why don't we just return an error tuple?
I think @arjan had some ideas about this.

Sent from my iPad

> On 29 aug. 2016, at 20:18, David de Boer <notifications@github.com> wrote:
> 
> In src/models/tests/m_rsc_db_tests.erl:
> 
> > @@ -11,8 +11,8 @@ modify_rsc_test() ->
> >      AdminC = z_acl:logon(?ACL_ADMIN_USER_ID, C),
> >      CatId = m_rsc:rid(text, C),
> >  
> > -    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}], C)),
> > -    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}, {category_id, CatId}], C)),
> > +    ?assertThrow({{error, eacces}, _Trace}, m_rsc:insert([{title, "Hello."}], C)),
> @mworrell I’m not sure this is what we want to return; wouldn’t just {error, eacces} be better?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>